ISL.Reidentification.Core/Brokers/Storages/Sql/Reidentifications/IReidentificationStorageBroker.AccessAudit.cs

```diff
@@ -0,0 +1,20 @@
+﻿// ---------------------------------------------------------
+// Copyright (c) North East London ICB. All rights reserved.
+// ---------------------------------------------------------
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using ISL.Reidentification.Core.Models.Foundations.AccessAudit;
+
+namespace ISL.Reidentification.Core.Brokers.Storages.Sql.Reidentifications
+{
+    public partial interface IReidentificationStorageBroker
+    {
+        ValueTask<AccessAudit> InsertAccessAuditAsync(AccessAudit accessAudit);
+        ValueTask<IQueryable<AccessAudit>> SelectAllAccessAuditsAsync();
+        ValueTask<AccessAudit> SelectAccessAuditByIdAsync(Guid accessAuditId);
+        ValueTask<AccessAudit> UpdateAccessAuditAsync(AccessAudit accessAudit);
+        ValueTask<AccessAudit> DeleteAccessAuditAsync(AccessAudit accessAudit);
+    }
```

Comment:
This is all you need for `DATA: `  as the methods will be implemented under `BROKERS:`

```suggestion
    public partial interface IReidentificationStorageBroker
    { }
```